packages/server/modules/workspaces/tests/unit/events/eventListener.spec.ts

```diff
@@ -15,17 +15,23 @@ describe('Event handlers', () => {
         {
           workspaceId,
           userId: cryptoRandomString({ length: 10 }),
-          role: Roles.Workspace.Admin
+          role: Roles.Workspace.Admin,
```

Comment:
So many different places needing this duplicate block of createdAt/updatedAt being added makes me feel like the type should've just been changed instead to make createdAt/updatedAt optional and then possibly renamed to WorkspaceAclInsert to represent the ACL model pre-insertion, and then possibly having another type with those fields made required whenever you're returning them from the DB.

I dunno, might be too big of a change so maybe not worth doing, but having to duplicate these dates everywhere does feel icky